### PR TITLE
Add infra destroy after go tests(ize-586) and copy md files to website/commands (ize-589)

### PR DESCRIPTION
**Pull request by kobrikx**
#### What's new: 

  - Added infra destroy after go tests

### CODE REVIEW

File: .github/workflows/run.e2e-tests.yml [hunk 1]
```diff
@@ -157,7 +159,9 @@ jobs:
       - name: Run Tests
         run: |
           go test -v --timeout 0 --tags="e2e bastion_tunnel" ./test-e2e
-          
+
+      - name: Destroy Infra
+        run: ize down infra
```
Comment (igorkotof) [inline .github/workflows/run.e2e-tests.yml:164]: @kobrikx have you tested it? if I'm not mistaken it should be like `ize down --auto-approve`

**Review by AutomationD — CHANGES_REQUESTED** [review]
Please make sure e2e tests run is successful.

**Review by igorkotof — APPROVED** [review]
lgtm now, thanks

File: .github/workflows/run.e2e-tests.yml [hunk 2]
```diff
@@ -99,6 +99,10 @@ jobs:
         run: |
           go test -v --timeout 0 --tags="e2e ecs_apps" ./test-e2e
 
+      - name: Destroy Infra
```
Comment (AutomationD) [inline .github/workflows/run.e2e-tests.yml:102]:
```suggestion
      - name: Cleanup Infra
```

**Review by AutomationD — APPROVED** [review]
Thank you, looks good.

**Pull request merged**